libraries/Microsoft.Bot.Builder.LanguageGeneration/Templates.cs

```diff
@@ -151,7 +162,17 @@ public class Templates : List<Template>
         public static Templates ParseFile(
             string filePath,
             ImportResolverDelegate importResolver = null,
-            ExpressionParser expressionParser = null) => TemplatesParser.ParseFile(filePath, importResolver, expressionParser);
+            ExpressionParser expressionParser = null)
+        {
+            var curTemplate = TemplatesParser.ParseFile(filePath, importResolver, expressionParser);
+            curTemplate.InjectToExpressionFunction();
+            foreach (var refer in curTemplate.References) 
```

Comment:
Personally we should move it into the constructor. It is the behavior of templates initialization, but not only the behavior of ParseFile.

Reply:
I tested the constructor. Found that, the injection should add to both constructor and parseFile (parseText also).